### PR TITLE
jq code

### PR DESCRIPTION
**Pull request by olidi10**
Hi Mary here is the code.Please let me know if you got it.
Thanks

### CODE REVIEW

**Comment by olidi10** [comment]
just reset my slack password for javascript.

File: js/script.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                        }, 500, "swing", function () {
+                            $li.remove();
+                            updateItemCount();
+
+                        });
+
+
+                    } else {
+                        $li.addClass("complete");
+                        list.append($li).hide().fade(300)
```
Comment (mbMosman) [inline js/script.js]: Look at how the variable for the list is defined at the top.  The variable name here is incorrect.

File: js/script.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                e.preventDefault(); // prevent the form from refreshing.
+                //get the text from the item list
+
+                var text = $("itemDescription").val();
+                var newItem = $("<li>" + text + "</li>");
+                $newItem.on("click", function () {
+                    var $li = $(this);
+                    completeitem($li);
+                });
+                $(list).append($newItem);
```
Comment (mbMosman) [inline js/script.js]: Look at how the variable for the list is defined at the top.  The variable name here is incorrect.

File: js/script.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+            function updateItemCount() {
+                var itemCount = $("li").length;
+                $("#counter").text(itemCount);
+
+            }
+            //call the function defined above to update the initial item count.
+            updateItemCount()
+                // Add ability to add a new item to the list
+            $("newItemButton").show();
+            $("newItemForm").hide();
```
Comment (mbMosman) [inline js/script.js]: The two values used in the jQuery functions on lines 20 & 21 are HTML id attribute values not tag names, so they are not specified correctly.

File: js/script.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+            // Build the new list item and add it when we click add.
+            $("#add").on("click", function (e) {
+                e.preventDefault(); // prevent the form from refreshing.
+                //get the text from the item list
+
+                var text = $("itemDescription").val();
+                var newItem = $("<li>" + text + "</li>");
+                $newItem.on("click", function () {
+                    var $li = $(this);
+                    completeitem($li);
```
Comment (mbMosman) [inline js/script.js]: JavaScript function names are case sensitive, so there is a typo here.

File: js/script.js [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+                        });
+
+
+                    } else {
+                        $li.addClass("complete");
+                        list.append($li).hide().fade(300)
+
+
+                    }
+                });
```
Comment (mbMosman) [inline js/script.js]: This line should only be a }

File: js/script.js [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+        }
```
Comment (mbMosman) [inline js/script.js]: This line should be }); and match to the very first line.

File: js/script.js [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+                $(this).addClass("complete");
+                var $li = $(this);
+                completeItem($li);
+            });
+            //Name our click handler function so we can add it to new items
+            function completeItem($li) {
+                // Need to see first if it was already complete
+                if ($li.hasClass("complete")) {
+                    $li.animate {
+                        $li.animate({
```
Comment (mbMosman) [inline js/script.js]: This line is repeated twice and should not be.

**Comment by mbMosman** [comment]
I've put in some hints to fix the errors.  Try to finish fixing things up on your own, and  then open a new pull request.

**Pull request closed without merging**